src/lib/app-helper.js

```diff
@@ -66,7 +66,12 @@ async function installPackages (dir, options = { spinner: null, verbose: false }
   return ret
 }
 
-/** @private */
+// Is this still used? it is exported, but there are no references to it -jm
+/**
+ * @param {string} scriptName  npm script name
+ * @param {string} dir directory to run npm script in
+ * @param {string[]} cmdArgs args to pass to npm script
+ */
 async function runPackageScript (scriptName, dir, cmdArgs = []) {
```

Comment:
if it's exported, usually i would say we can't remove it without bumping a major, but considering it's an app helper script (internal)I think it would be fine to remove.